Add `_next` property to the Select interface type for UNION queries

**Describe the bug**
When SQL Select statements use a **UNION** operator, the generated AST has a `_next` property that contains the ASTs for the other tables being unioned. However this property is not exposed in the [Select interface type](https://github.com/taozhi8833998/node-sql-parser/blob/465a89108dd9694ffecc6f9c3f5545757d763190/types.d.ts#L222) so the typechecker fails when trying to access those nodes. 

**Database Engine**
Snowflake

**To Reproduce**
**node:** 18.20.1
**node-sql-parser:** 5.3.2

```typescript
import { Parser } from "node-sql-parser";

const parser = new Parser();
const opts = { database: "Snowflake" };

const query = `
        SELECT user_id, timestamp
        from login.account.password_reset
        union
        select user_id, timestamp
        from "LOGIN"."ACCOUNT"."PASSWORD_UPDATED"
        union
        select user_id, timestamp
        from LOGIN.ACCOUNT.SUCCESSFUL_LOGIN;
`;

const { ast } = parser.parse(query, opts);
const firstUnionTable = ast[0]._next
```

```
error TS2339: Property '_next' does not exist on type 'Select'
```


**Expected behavior**
should be able to access `_next` without typescript throwing an error

--- a/types.d.ts
+++ b/types.d.ts
@@ -233,6 +233,7 @@ export interface Select {
   limit: Limit | null;
   _orderby?: OrderBy[] | null;
   _limit?: Limit | null;
+  _next?: Select | null;
   parentheses_symbol?: boolean;
   _parentheses?: boolean;
   loc?: LocationRange;

